frontend/components/buttons/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from 'lib/jest'
+import { Button } from '../Button'
+
+describe('<Button />', () => {
+  it('shows the passed label', () => {
+    render(<Button label="This is a button" />)
+
+    expect(screen.getByRole('button', { name: /this is a button/i })).toHaveTextContent(
+      'This is a button'
+    )
+  })
```

Comment:
I think another test worth adding here is to test if `onClick` function is called if button is clicked

But personally - I don't see much value in this test. It's like we are testing the behavior of an HTML element. I see more value testing components that have more logic in it 🤔 What do you think?

Reply:
Actually, Since this PR is more on the UI side, Little to nothing to test pa. 
I can do the `onClick` test, but as you said, it's like testing the HTML element behavior.